### PR TITLE
Fix page respoinsive, refactoring components

### PR DESCRIPTION
**Pull request by sashachabin**
- Remove horizontal scroll on phone screens
- `<figcation>` "Экосистема Екатеринбурга"  → `<h2>` in `<main>`

**+ Refactoring**
- Move global typography to Page.module.css
- Constistence media-query. `@media max-width` → `@media min-width`
- Remove duplicating CSS rules (e.g. dobule line-height)
- Separate `<Road>` and `<Ecosystem>` components

### CODE REVIEW

File: src/components/action/FooterAction.module.css [hunk 1]
```diff
@@ -14,9 +13,9 @@
   transform: scale(1.08);
 }
 
-@media screen and (max-width: 768px) {
+@media screen and (min-width: 768px) {
```
Comment (sashachabin) [inline src/components/action/FooterAction.module.css:16]: I swaped rules max-width -> min-width

File: src/components/preview/Preview.module.css [hunk 2]
```diff
@@ -15,6 +17,7 @@
 
   @supports (height: 1svh) {
     .preview__graphics {
+      display: inline-block;
       margin-top: 0;
       height: 300dvh;
```
Comment (sashachabin) [inline src/components/preview/Preview.module.css:22]: `.preview__graphics` is inline (`<svg>`  tag) and `height` rule not works.


I add `display: inline-block`. But, in fact, we can delete it)

image

File: src/components/page/Page.module.css [hunk 3]
```diff
@@ -1,21 +1,49 @@
...
+    display: block;
+    overflow: hidden;
+    margin: 0;
+    padding: 0.6em 0.2em;
+    color: white;
+    font-weight: 300;
+    line-height: 1.375;
+  }
+  
+  & ul {
```
Comment (gcor) [inline src/components/page/Page.module.css]: [optional] we don't have normalizer, looks like we can return it to global css

File: src/components/header/Header.module.css [hunk 4]
```diff
@@ -44,8 +41,6 @@
   color: #fff;
   text-decoration: none;
   transition: all .15s cubic-bezier(0.52, 0.51, 0.39, 0.99);
```
Comment (gcor) [inline src/components/header/Header.module.css:43]: could we use yellow btn on mobile too?
Screenshot 2023-04-03 at 22 05 16

**Review by gcor — APPROVED** [review]

**Pull request merged**